common/current_preset.h

```diff
@@ -271,10 +271,30 @@ class CurrentPreset {
 
   void SetPreset(int preset) {
     Clear();
+    #ifdef SAVED_PRESET
```

Comment:
I don't like using #define SAVED_PRESET for this, it's not not descriptive enough.
Maybe something like:

#define SAVE_STATE
or
#define SAVE_CURRENT_STATE
or
#define SAVE_PRESET_AND_VOLUME

or, we could make it two defines:
#define SAVE_ACTIVE_PRESET
#define SAVE_VOLUME

Reply:
ok, moving to SAVE_STATE